modules/job/sqoop/config/sqoop.xml

```diff
@@ -27,6 +27,7 @@
 				<value>fs.defaultFS=${fsUri}</value>
 				<value>yarn.resourcemanager.address=${resourceManagerHost}:${resourceManagerPort}</value>
 				<value>yarn.application.classpath=${spring.hadoop.yarnApplicationClasspath}</value>
+				<value>mapreduce.framework.name=yarn</value>
```

Comment:
should this always be `yarn` or we want to make it configurable. From the apache docs, I see it could take one of local, classic or yarn as the value.